lib/py/images/images.py

```diff
@@ -172,7 +172,7 @@ def count(self, f):
 
         while True:
             buf = f.read(4)
-            if buf == '':
+            if len(buf) == 0:
```

Comment:
Alternatively we can use `b' '` on the right side of the assignment to have similar code to: 841edaf31 ("lib: a few fixes to be compatible with python3").

Also shouldn't we also update all other `if buf == ' ':` places:
ipc_msg_queue_handler:load
ipc_msg_queue_handler:skip

Reply:
Thank you Pavel for the code review!

I've updated the pull request to fix all other places as well. IMHO, `len(buf) == 0` is more readable and reliable solution for this problem (e.g., `buf == ' '` is not the same as `buf == ''`)

Reply:
Sorry for `b' '`, this was a misprint, I meant `b''` =) And obviously you are right - len check is more readable.